base/sysimg.jl

```diff
@@ -360,10 +360,10 @@ using .I18n
 
 # frontend
 include("initdefs.jl")
-include("Terminals.jl")
-include("LineEdit.jl")
-include("REPLCompletions.jl")
-include("REPL.jl")
+include("repl/Terminals.jl")
```

Comment:
We should probably introduce Path types and have `path"repl/Terminals.jl"` produce whatever the native path syntax is. That would also allow `path"C:\some\where"` to work on Windows systems without double backslashes.